\chapter{Lecture Exercises}
\setlength{\headheight}{22.94003pt}
\addtolength{\topmargin}{-10.22661pt}

\section{02.09.24 - Exercise 1 - CasePCR}
A part of the DNA fragment shown below is to be amplified using PCR and the primers P1 and P2. Following amplification the PCR product is cut with the restriction enzymes EcoRI (recognising the sequence 5'- G A A T T C -'3 and cutting with a 4-base overhang).

The DNA fragment:

5'-	A G	\underline{$C A G C_{P1}$}	G T	T T	G A	A T	T C	G A	C T	T C	A C	T T	C A	-'3

3'-	T C	G T	C G	C A	A A	C T	T A	A G	C T	G A	\underline{$A G T G_{P2}$}	A A	G T	-'5

The primers:

P1: 5’ C A G C 3’	

P2: 5’ G T G A 3’

\subsection*{What is the size (in base pairs) of the PCR-product?}
The P1 primer starts binding at the $3{rd}$ base pair from the left, marking the start of the PCR product and is included in the count. The P2 primer binds at the $21^{st}$ base pair, marking the endpoint of the PCR product, which is also included.

The PCR product will therefore be the DNA sequence from the 3rd to the 21st base pair. To find the total number of base pairs within this range, we wil use Equation \ref{eq:PCRProductSize}.

\begin{equation}
    \text{PCR product length} = (\text{End position}) - (\text{Start position}) + 1
    \label{eq:PCRProductSize}
\end{equation}

Now we can insert the values of the PCR product and the length can therefore be calculated from Equation \ref{eq:PCRProductSize1.1}:

\begin{equation}
    21 - 3 + 1 = 19
    \label{eq:PCRProductSize1.1}
\end{equation}

5'- C A G C G T T T G A A T T C G A C T T -'3

3'- G T C G C A A A C T T A A G C T G A A -'5

\subsection*{How many copies of DNA do you have after 4 PCR cycles if you start with 3 copies of the DNA fragment to be copied (under ideal conditions)?}
Since PCR follows an exponential growth model, the amount of DNA doubles with each cycle, represented by $X \cdot 2^n$, where $X$ is the initial number of DNA copies, and $n$ is the number of PCR cycles. After 4 PCR cycles, the DNA quantity will be $2^4 = 16$ times the initial amount.

Given that we start with 3 copies of the DNA fragment, the total amount of DNA after 4 PCR cycles will be $3 \cdot 2^4 = 48$ copies.

\subsection*{What is the size of the 2 fragments after cutting with EcoRI?}
From the first task, we know that the PCR product is 19 base pairs long. The EcoRI restriction enzyme recognizes the sequence 5'-G A A T T C-3' and cuts with a 4-base pair overhang. The cutting site of EcoRI is 
(
    5'-G A A 
\begin{tikzpicture}[baseline={(0,-0.5ex)}]
  \draw[decorate, decoration={zigzag, segment length=4, amplitude=2}] (0,0) -- (0,1.5ex);
\end{tikzpicture}
T T C-3'
). 

\vspace*{0.5em}
The PCR product is:

5'- C A G C G T T T \textcolor{red}{G A A T T C} G A C T T -'3

\vspace*{0.5em}
The cut will be between the $11^{th}$ and the $12^{th}$ base pair, and the two fragments will be:

Fragment 1: 5'- C A G C G T T T \textcolor{red}{G A A} -'3 = 11 base pairs.

Fragment 2: 5'- \textcolor{red}{T T C} G A C T T -'3 = 8 base pairs.

\section{11.09.24 - Exercise 1 - Replication, Transcription and Translation in prokaryotes}
For an automated version of answering this exercise, please see the file w2e1.py in the Python folder. This file will automatically replicate, transcribe and translate the given DNA sequence. Use the file as a template for future exercises if needed.

Figure \ref*{fig:Exercise1Seq} shows a DNA sequence which will used in Exercise 1 and its sub tasks. The given DNA sequence can be seen in figure \ref{fig:Exercise1Seq}

\begin{figure}[h]
    \centering  
    \includegraphics[width=1\textwidth]{Figures/Exercise1Seq.png}
    \caption{Sequence of DNA which is used in Exercise 1}
    \label{fig:Exercise1Seq}
\end{figure}
\subsection{Replication in prokaryotes}
Replicate the DNA sequence based on the anti-sense (template) strand written below using the following primer \textbf{5’- AUCCAUU -3’} (This RNA complementary to the DNA sequence and is the defined -35 region, which will be used in the transcription subsection).

\vspace{1em}
\begin{itemize}
    \item While replicating the DNA, identify the leading and lagging strands. Furthermore remember that G and C are the most stable base pairs, A and T are the least stable base pairs
    \item The DNA sequence is replicated from the 3’ to 5’ direction
    \item Use the primer as to define where to start from
\end{itemize} 

\vspace{1em}

5'- X X X \textbf{A T C C A} \textbf{\sethlcolor{Turquoise}\hl{T T G A C A}} C T A A G C A T G C A T T G C A C T G C \sethlcolor{green}\hl{\textbf{T A T A A T}} C G T A G G T G G C G T \textbf{\sethlcolor{Apricot}\hl{A T G}} T C T G C T A T A C G T A T C G A C T A G T A \textbf{\sethlcolor{Bittersweet}\hl{T A G}} C G G -3'      (This is the replicated)

\vspace{1em}

3’- \textit{G A T} \sethlcolor{yellow}\hl{\textbf{T A G G T A A}} C T G T G A T T C G T A C G T A A C G T G A C G A T A T T A G C A T C C A C C G C A T A C A G A C G A T A T G C A T A G C T G A T C A T A T C G C C-5’      (This is the given)

\vspace{1em}

So, to begin with, we will look at the given RNA primer, and convert it to a DNA primer.
5'-AUCCAUU-3' will be converted to 5'-ATCCATT-3' (since RNA has Uracil instead of Thymine). 


Afterwards we will look at the given DNA sequence and find the complementary strand to it. The complementary strand will be the anti-sense strand. 5'-ATCCATT-3' will be converted to 3'-TAGGTA-5' (since A pairs with T, T pairs with A, C pairs with G, and G pairs with C).


Now we will highlight the given sequence in \hl{yellow} and start translating the DNA sequence from the 3' to 5' direction. We will start from the primer and move towards the end of the sequence.

\vspace{1em}
This concludes this part of the exercise.

\subsection{Transcription in prokaryotes}
Identify the Pribnow (-10 region) and (-35) regions in the above DNA double-helix (sense strand). Transcribe DNA into a Messenger-RNA (mRNA).

\vspace{1em}
In this course the Pribnow box (-10 region) is defined as the region 5’-TATAAT-3’ and the -35 region is defined as 5’-TTGACA-3’. Wil will therefore 

We found the Pribnow box sequence and highlighted it in \sethlcolor{green}\hl{green}. The mRNA strand will be transcribed from the anti-sense strand, so we will use the antis sense strand to transcribe the mRNA until the -35 region which we highlighted in \sethlcolor{Turquoise}\hl{turquoise}.


We can now go to the right end (10-ish letters) of the pribnow box and start transcribing the mRNA. In this course, the start codon is defined as the first AUG codon after the Pribnow box. The mRNA strand will be transcribed from the 3' to 5' direction.

The start codon is highlighted in \sethlcolor{apricot}\hl{apricot} starting from the 13\textsuperscript{th} letter to the right. The stop codon is highlighted in \sethlcolor{Bittersweet}\hl{bittersweet}.


The Pribnow 10 region is \textbf{5’- ATCCATT -3’} and the -35 region is \textbf{5’- CGATAGG -3’}.

Now that we have identified both the start- and stop codon, we can start to transcribe the mRNA strand. The first codon will start from number 10 after the the Pribnow box. The mRNA strand will therefore be: 

\sethlcolor{CarnationPink}\hl{5'- GUA UGU CUG CUA UAC GUA UCG ACU AGU AUA GCG -3'}

\subsection{Translation in prokaryotes}
Translate the mRNA strand from transcription step into amino acids.

\vspace{1em}
The mRNA strand is written in the last subtask, and have already been translated into codons. It's now possible to translate the codons to amino acids.

H\textsubscript{2}N- Val - Cys - Leu - Leu - Tyr - Val - Ser - Thr - Ser - Iso - Ala -COOH

\section{11.09.24 - Exercise 2 - PCR and Restriction Enzymes}
A part of DNA fragment shown below is to be amplified using PCR and the primers F and R.  Subsequently the PCR product is cut with restriction enzyme HpyCH4V and the resultant products are run on a gel. The

\begin{figure}[h]
    \centering
    \includegraphics[width=1\textwidth]{Figures/Exc2DNA.png}
    \caption{Sequence of DNA which is used in Exercise 2}
    \label{fig:Exercise2Seq}
\end{figure}

The two given primers are:
\begin{itemize}
    \item Primer F: 5’- AACCT-3’
    \item Primer R: 5’-TGGAT-3’ (3'-TAGGT-5')
\end{itemize}

\subsection{What is the size (in base pairs) of the PCR-product?}
The Primer F starts from the 3' end of the DNA sequence, and the Primer R starts from the 5' end of the DNA sequence. The PCR product will therefore be the DNA sequence between the two primers.

Primer F has to fit to the upper strand, and Primer R has to fit on the lower strand. The primners sequence are therefore:
\begin{itemize}
    \item Primer F: 5'-AACCT-3' 
    \item Primer R: 5'-TGGAT-3' -> 3'-TAGGT-5'
\end{itemize}

Primer F starts from the 15\textsuperscript{th} to the right and ends at the 19\textsuperscript{th} base pair. Primer R starts from the 64\textsuperscript{th} from the right and ends at the 68\textsuperscript{th} base pair. The PCR product will therefore be the DNA sequence between the 19\textsuperscript{th} and 64\textsuperscript{th} base pair.

\underline{The size of the PCR-product is therefore 68-14 = 54 base pairs.}

\subsection{How many copies of DNA do you have after 4 PCR cycles if you start with 3 copies of the DNA- fragment to be copied (considering ideal conditions)?}
After each PCR cycle, the amount of DNA doubles. After 4 PCR cycles, the amount of DNA will be 2\textsuperscript{4} = 16 times the original amount of DNA.
Though, we are starting with 3 copies of the DNA fragment, so the amount of DNA after 4 PCR cycles will be $3 \cdot 2^4$ = 48 copies of the DNA fragment.

\subsection{What is the size of 2 fragments after cutting with HpyCH4V?}
The restriction enzyme HpyCH4V recognizes the sequence 3'-ACGT-5'. The DNA sequence is cut at the recognition site, and the two fragments will be the DNA sequence before the recognition site and the DNA sequence after the recognition site.

5'- TGTCGGTGTG CCATAACCTA AGTCATAACG ATCCGTGAAT GCATACCCGT TTTGAATAAT TGAATCCAAG AAACTTGTAT CTGTACTTGA-3'

3'- ACAGCCACAC GGTATTGGAT TCAGTATTGC TAGGCACTTA CGTATGGGCA AAACTTATTA ACTTAGGTTC TTTGAACATA GACATGAACT-5'
\vspace{1em}

HpyCH4V will therefore cut the DNA sequence at number 41, and the two fragments will be:

5'- AACCTA AGTCATAACG ATCCGTGAA\textbf{T GCA}TACCCGT TTTGAATAAT TGAATCCA-3'

3'- TTGGAT TCAGTATTGC TAGGCACTT\textbf{A CGT}ATGGGCA AAACTTATTA ACTTAGGT-5'

\vspace{1em}   
\underline{There will therefore be 27 base pairs in the first fragment and 27 base pairs in the second fragment.}

\subsection{Place the bands that you would observe after running your product on the an agarose gel (path 1 before cutting with enzyme, path 2 after cutting)}

When you run the PCR product on a gel electroforesis, you will see the DNA fragments separated based on their size. The smaller fragments will move faster through the gel than the larger fragments. This means that both the DNA fragment before and after cutting with the restriction enzyme will be visible on the gel since they doesn't have the same fragment size   .

Therefore you will see two bands on the gel, one at 54 base pairs (before cutting) and one at 27 base pairs (after cutting).
This can be seen in Figure \ref{fig:Exercise2Gel}.
\clearpage
\begin{figure}[t]
    \centering
    \includegraphics[angle=270, width=0.35\textwidth]{Figures/Exc2Gel.JPG}
    \caption{Gel electrophoresis of the DNA fragments and the PCR products' bands}
    \label{fig:Exercise2Gel}
\end{figure}

\section{11.09.24 - Exercise 3 - Primer3plus}
Based on the sequence below design primers that fulfill following conditions:

The following information is given:

\begin{highlight}
    \begin{itemize}
        \item size (22-24bp)
        \item Tm (55-57)
        \item GC\% (35-70)
        \item amplicon size (300-400bp)
    \end{itemize}
\end{highlight}

The following sequence is given:

\begin{verbatim}
    TTGCAAGCCCTGATTTAAGCATTACACCAGCATCATCAAAGTAATACGCATTTCCATTTTGATCAGTTAT
    ATATTGGTCAAAAACTTGATGACCTTTCTTATCGAAATAAACAATTGTACCATCTTCGTTTTGTAAGAAG
    CTCTTGACCAGTTCGATACCATTAGGTAAGAAGAAGTAGGTATGGTTATTAATCTTTTGCAAACCTGTTA
    CTAAATGACCTGTTTTATCAAAATAGTAATAGTTATTATCATCATCTTGAATAAATGCATTTTGGGCACG
    ATAACCACTTAATGTATAATAGATGATTCCTTTATCATCGTGTGTAAAGCCAGTTTCTGACAGATCATTA
    GTTAACTGTTTTGGTAAGTAGTCACCATCCTCAGTGTTCGAAACGACCTTAAAGTACTTATCAGAACCCA
    TATCTTTCAATACGTATCCAGCACCTTTACCTTGGATGTTAGAGCCATTGAAGTACTTAGCCGCCCACTC
    AGTAATCTTTACACTGCCATCAATTGGAACGCCAGTTGAGATTTGATTCACTTTAAATAGGGATGGATAG
    AGTGCCTGTAACTCTTCTAAGAAGGCACCACCATACATCTCTTGATATTGACCACCCCCACGACTTTGTA
    CAACATATAAGGCATTGTCAATATCAGAATCTGTATCGTCATCTCCAAATGAATTTGTTCTTGTGACAGT
    AGCTAATTCTTGCTCTGGCAAATTATAAATTTGGTCCGGCACCCAATCGGCAATGGCTTGAATACCGCTA
    GCATGTAAGGCTTTAATAGCATCGCGCAACTGATCAGCAGTTCCATATTTTGTCGGTGTGCCATAACCTA
    AGTCATAACGATCCGTGAATGCATACCCGTTTTGAATAATTGAATCCAAGAAACTTGTATCTGTACTTGA
    ACGATATTGTGGTGCCAATTGGAAGCTTGTCACACCCCATTGCTTAAATTGGTCCGCATTCTGAGCGATG
    ACTACGTTTGTATATTCACTGCTGTCTGTAGCAAATGCTTGGAAGTTTGAGAAACCTTCGTAGATGACTT
    GAGAATCAAGAGCAGCGTTTGAATGGAACACTTTATCACTCGTGTTTGTTGTTGTATCAGAGGCCGTTCG
    TGCATCTTGATCTTGTTGCGCACCTACCGGAACCCAAACTGCCAAGTAACCAGAAACTTGTGGATTTTGT
    ACACCATAAATTGATTCATTCGTAAAAATCAAATCGCCGTTAGCATCTGTGTACGCCACAGGTGCATTTT
    CATCAGTATCATAATAAGCTAATCCATCTGCAGTTGTTGATAACAAAGCACGATAAGCTTGGTTCTTATG
    AGCTGCCCCCATATGCAATGTGACAGTATGCCCATCCTCTAATTGTAGCTCCGCGTTATTGCTGACGATG
    ACTCCAATACCTTCAGTACGCGTCTCAGATGTTCCAGTGTCAGAAGCCGTCATGGCATCTTTACCATAGC
    GAACACTTGTTAACACGTCATTACTATCAACGGACATCGATTGGCCACCAGCAACATATTGGACTCTAGC
    CTTCAGCAAAGTGTTAATCGCATCATAGTATGGTGACTTTGTTGCCATATATTGACCATCATCTGTATAT
    AAATCGCCATAATAGACACGAGGAACAGTATCCTTGTTGGTTAGCAACATCGCATAAGCACTAGCCATAT
    TATATTGTGTGTACTTTTTGTCTGCTAATTTTTCATCTTCATTGTATACTTTGAAAGCAGCTGCCAATTG
\end{verbatim}


\vspace{1em}
Use Primer3Plus (http://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi/)

\vspace{1em}
While setting the conditions, it's important to remember that the amplicon size is the size of the DNA fragment that will be amplified by the primers.This is named "Product range size" on the web-side under general settings.

\vspace{1em}
Write answers in the table bellow:
\begin{table}[h]
    \centering
    \caption{The given table which should be filled out with the information from the Primer3Plus web-side}
    \label{tab:Exercise3PrimerTable}
    \begin{tabular}{c|c|c|c|c}
        \textbf{Primer} & \textbf{Sequence (5'-3')} & \textbf{Primer size (bp)} & \textbf{Tm (°C)} & \textbf{GC content\%} \\
        \hline
        Forward & TCAGTAATCTTTACACTGCCATC & 23 & 56.1 & 39.1 \\
        Reverse & TTATGACTTAGGTTATGGCACAC & 23 & 56.1 & 39.1 \\
    \end{tabular}
\end{table}

The second part of this exercise was to use this information to deduct which bacteria this sequence belongs to. The sequence is from the bacteria \textit{Leuconostoc mesenteroides}.

\section{11.09.24 - Exercise 4 - Primers check and restriction enzymes}
The following information is given:
\begin{itemize}
    \item Primer F: 5’- GGAACCCAAACTGCCAAGT-3’
    \item Primer R: 5’-CCTCGTGTCTATTATGGCGATT-3’
\end{itemize}

Using \textbf{“In silico PCR amplification”} (http://insilico.ehu.es/PCR/) answer following questions:

\subsection{What \textit{Leuconostoc} strain will the primers align to?}
We start by going to the web-side. Then we choose \textit{Leuconostoc} and press "Next step". Afterwards we paste the primer sequences into the "Forward primer" and "Reverse primer" fields. We then choose "APPLY TO ALL Leuconostoc" and press "Amplify".

Then we can see 2 bands where the primers has traveled at 485 and 485 base pairs. The primers will therefore align to the 2 \textit{Leuconostoc mesenteroides} strains.

\subsection{What would be the PCR product size (bp) if we use those primers against that species?}
Based on the bands that was produced the PCR product size will be 485 base pairs.

\vspace{3em}
For the next subsections, the following information is given:

Using NEBcutter V2.O  http://tools.neb.com/NEBcutter2/ retrieve the amplicon from “In silico PCR amplification” and answer following questions:

\subsection{What will be the size of DNA products after digesting the amplicon with Hind III restriction enzyme?}
To use the web-side you need to copy the DNA sequence from the "In silico PCR amplification" web-side and paste it into the "Enter DNA sequence" field and press sumbit. Then you look at the sumary and find the restriction enzyme HendIII and hold the mouse over it to see where it cuts. 

\vspace{1em}
For the two \textit{Leuconostoc} strains the HINDIII cuts at:
\begin{itemize}
    \item Cuts at bp 167 (Leuconostoc mesenteroides subsp. mesenteroides ATCC 8293)
    \item Cuts at bp 167 (Leuconostoc mesenteroides subsp. mesenteroides J18)
\end{itemize}

\vspace{1em}
This means that the DNA products will be 167- and 318 base pairs long after digesting the amplicon with HindIII restriction enzyme. This makes sense since the total sequence, for both, is 485 base pairs long. Thus 167+318 = 485 base pairs.

\subsection{What are the reaction conditions requires for ApeKI?}
To answer this, the following table has to be filled out. The information can be found on the NEBcutter web-side by pressing "Enzyme list" and then "ApeKI". Then in the bottom right corner of the first table, there is a button called "NEV Restriction Enzyme - Activity/Performance Chart". Then find the enzyme ApeKI and press on the buffer, there all the needed information is found. 

\begin{table}[h]
    \centering
    \caption{The given table which should be filled out with the information from the NEBcutter web-side}
    \label{tab:Exc4ApeKI}
    \begin{tabular}{c|c}
        \textbf{Buffer} & \textbf{NEBuffer\textsuperscript{TM} r3.1} \\
        \hline
        Salt & 100 mM NaCl \\
        Main & 50 mM Tris-HCl \\
        pH & 7.9 at 25\textdegree C  \\
        Mg & 10 mM MgCl\textsubscript{2} \\
        rAlbumin & 100 \textmu g/ml Recombinant Albumin\\
        Reaction temp. (\textdegree C) & 75\textdegree C (Obtained from the solutions)\\
    \end{tabular}
\end{table}

\subsection{What is the digestion pattern of AciI?}
The AciI enzyme cuts at the following base pairs:

Primer F 5'-...AA\textsuperscript{$\downarrow$}CGTT...-3'

Primer R 3'-...TTGC\textsubscript{$\uparrow$}AA...-5'

This information can be found on the page before you press "NEV Restriction Enzyme - Activity/Performance Chart".

\subsection{Using ENDMEMO}

For this exercise we need to use this web-side (http://www.endmemo.com/bio/egel.php)
In the graph on the right place bands you would  receive after digesting the amplicon with HindIII, ApeKI and AciI in one reaction. 

Here we simply need to insert a sequence and choose the right restriction enzymes. The bands will then be shown on the gel.

\begin{itemize}
    \item HindIII cuts at 167 bp; bands will be at 167- and 318 bp
    \subitem Blue color
    \item ApeKI cuts at 184 bp; bands will be at 184- and 301 bp
    \subitem Green color
    \item AciI cuts at 233 bp; bands will be at 233- and 252 bp
    \subitem Red color
\end{itemize}

\begin{figure}{h}
    \centering
    \includegraphics[width=0.35\textwidth]{Figures/Exc4sub6.png}
    \caption{Gel electrophoresis of the DNA fragments and the PCR products' bands}
    \label{fig:Exc4sub6}
\end{figure}

\section{11.09.24 - Exercise 5 - Translation}
The mRNA is a type of RNA that “passes” genetic information from DNA to protein. In this process (called translation) mRNA is read on ribosomes according to the genetic code. Below you can find an example of mRNA sequence:

\vspace{1em}

A A \colorbox{green}{U A G} G G A U G G G A U U U A U G A G A A A A G C C G A U U U U G A G C U U U A U C G U G A U G C A G A U A A A C A U U A U A A U C A G G U A C U A A C C A C A C G U G A U U U U C U U G C A C U A G G U G U U G G A A C A A U U A U U U C A A C U U C U A U C U U U A C C U U G C C A G G A C A A G U U G C C G C U C A A U U U G C G G G C C C A G G G U U G U C U U U U C U U A U C U A C U U G C C G C U U U G G U C G C U G G U U U U G U G G C C U U G G C U U A U G C U G A A A U G U C A A C A G U A A U G C C C U U U G C U G G U U C A G C C U A U U C U U G G A U U U C G G U C U U A U U U G G C G A A G G U U U C G G C U G G A U U G C U G G U U G G C U U U G C U U G C U G G A A U A U U U A U U G C U G U U G C C U U U G U U G G C U C A G G U U U C U C A G C A A A U C U U C A G C A A U U A C U A G C G C C A C U C G G U U U C C A A C U G C C A A A G U A U U G G C C A A C C C U U U G G U A C A G A U G G U G G A U C G U U G A U A U A U U C A C U U U G G U C A U U U G C U C U C G G C A A U U A U U G U C U U U C G U G G G C A U C U G A U G C U G G A C G A A U C A G U C A A U U C U U G U U G U C U U A A G U G G C A G C G G U U A U U G C U U U C A U C A U U G G G A U U A C G G U C A U U A A G C C A G C A A A U U A C C A U C C A U U U A U U C C G C C A C A C A A U C C G A A A C A G G U U U G G U G C U U C U C G G G A U U U G G U C U G G G U G U C G A U G A U U U C U U G G C C U A U A U C G G U U U G A U U C A U U G C A G C C A A U U C A G C G G A A G C U A A A A U C C C A A A A C A U G C C A C G U G G A A U U A U U G G G U C G C U U C A U U G C U G U U G U U C U A U U G C A G C A G U A A C C U U G G U A C U U G U A G G A U G C A U C C U U A C U C A G C C U A U G C A G G A A A U G C A G C G C C A G U U G G U U G G C C C U U C A A U C A G G U U A U U C U G U U U U A U C A G A A G U U G U C C A G C G A U G C A U G G A C U G G A U U C A A G C A A A A A C U U U A U U C C G C A A U G C C A A A A \colorbox{red}{U A A} G G C C U


\vspace*{1em}
Using provided sequence and some online bioinformatics tools try to find answers for the following questions:

\subsection*{Using a nucleotide translation tool, (e.g. http://web.expasy.org/translate/ ) determine in which frame (1,2 or 3) and which direction (5' to 3' or 3' to 5') is this mRNA transcript given?}
When inserted into the web-side, the mRNA sequence is translated into a nucleotide (DNA/RNA) sequence. The sequence is translated into a protein sequence, and the frame and direction is given. 

The most likely frame and direction is 5' to 3' in frame 2, which is marked most in red and illustrates the most open reading frame.

\subsection*{Underline the start codon and the stop codon on the sequence.}
The start- and stop codon has been marked in the sequence above. 

The start codon is the first AUG codon in the sequence (highlighted in green). The stop codon is the las UAA, UAG or UGA codon (highlighted in red). The fastest way to find this is by searching with ctrl+F in the word document. 

\section{18.09.24 - Exercise 1 - NCBI BLAST}
This part of the exercise will take you through some uses of the NCBI BLAST homepage. The NCBI stands for National Center for Biotechnology Information and composes of multiple databases relevant in biotechnology. It was founded in 1988 and since that time is constantly growing. The DNA and protein databases are immense however they may contain unverified and unreliable information. Therefore it is important to carefully interpret results.

\subsection*{Go to the NCBI BLAST homepage http://blast.ncbi.nlm.nih.govBlast.cgi}
-Use the sequence bellow to explain differences between blastn, blastp and blastx

AATAGGGATGGGATTTATGAGAAAAGCCGATTTTGAGCTTTATCGTGATGCAGATAAACATTATAATCAGGTACTAACCAC


The blastn and blastp were not able to find any hits, only error codes came. The blastx was able to find various hits. The reason for this is that the blastn is used for nucleotide sequences, blastp is used for protein sequences and blastx is used for translated nucleotide sequences. The blastx is therefore able to find hits since it translates the nucleotide sequence into a protein sequence.

\subsection*{Copy paste the fasta sequence below, seq1 (16S rRNA\_F18\_Gergoush), and choose the relevant BLAST tool. Do the blast using default settings.}
>seq1 (16S rRNA\_F18\_Gergoush)

\begin{verbatim}
    AGATGGGAGCTTGCTCCCTGATGTTAGCGGCGGACGGGTGAGTAACACGTGGGTAACCTGCCTGTAAGAC
    TGGGATAACTCCGGGAAACCGGGGCTAATACCGGATGGTTGTTTGAACCGCATGGTTCAAACATAAAAGG
    TGGCTTCGGCTACCACTTACAGATGGACCCGCGGCGCATTAGCTAGTTGGTGAGGTAACGGCTCACCAAG
    GCAACGATGCGTAGCCGACCTGAGAGGGTGATCGGCCACACTGGGACTGAGACACGGCCCAGCACTCCTA
    CGGGAGGCAGCAGTAGGGAATCTTCCGCAATGGACGAAAGTCTGACGGAGCAACGCCGCGTGAGTGATGA
    AGGTTTTCGGATCGTAAAGCTCTGTTGTTAGGGAAGAACAAGTACCGTTCGAATAGGGCGGTACCTTGAC
    GGTACCTAACCAGAAAGCCACGGCTAACTACGTGCCAGCAGCCGCGGTAATACGTAGGTGGCAAGCGTTG
    TCCGGAATTATTGGGCGTAAAGGGCTCGCAGGCGGTTTCTTAAGTCTGATGTGAAAGCCCCCGGCTCAAC
    CGGGGAGGGTCATTGGAAACTGGGGAACTTGAGTGCAGAAGAGGAGAGTGGAATTCCACGTGTAGCGGTG
    AAATGCGTAGAGATGTGGAGGAACACCAGTGTCGAAGGCGACTCTCTGGTCTGTAACTGACGCTGAGGAG
    CGAAAGCGTGGGGAGCGAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGATGAGTGCTAAGTGT
    TAGGGGGTTTCCGCCCCTTAGTGCTGCAGCTAACGCATTAAGCACTCCGCCTGGGGAGTACGGTCGCAAG
    ACTGAAACTCAAAGGAATTGACGGGGGCCCGCACAAGCGGTGGAGCATGTGGTTTAATTCGAAGCAACGC
    GAAGAACCTTACCAGGTCTTGACATCCTCTGACAATCCTAGAGATAGGACGTCCCCTTCGGGGGCAGAGT
    GACAGGTGGTGCATGGTTGTCGTCAGCTCGTGTCGTGAGATGTTGGGTTAAGTCCCGCAACGAGCGCAAC
    CCTTGATCTTAGTTGCCAGCATTCAGTTGGGCACTCTAAGGTGACTGCCGGTGACAAACCGGAGGAAGGT
    GGGGATGACGTCAAATCATCATGCCCCTTATGACCTGGGCTACACACGTGCTACAATGGACAGAACAAAG
    GGCAGCGAAACCGCGAGGTTAAGCCAATCCCACAAATCTGTTCTCAGTTCGGATCGCAGTCTGCAACTCG
    ACTGCGTGAAGCTGGAATCGCTAGTAATCGCGGATCAGCATGCCGCGGTGAATACGTTCCCGGGCCTTGT
    ACACACCGCCCGTCACACCACGAGAGTTTGTAACACCCGAAGTCGGTGAGGTAACCTTTTAGGAGCCAGC
    CGCCGAAGG    
\end{verbatim}

Again, the blastn and blastp were not able to find any hits, only error codes came. The blastx was again able to find various hits. 

\subsubsection*{When you get the result discuss with your group: How is the identification result presented, what information do you get?}
The information given from the blast is, description, scientific name, max score, total score, query cover, e value, per. identified, accesion length and accesion number.

The description gives a brief summary of the hit, the scientific name gives the name of the organism, the max score reflects the highest alignment score of the hit, the total score is the cumulative score of the hit, the query cover is the percentage of the query that is covered by the hit, the e value is the expectation value lower e values are good (e values under 0.01 are considere significant). The per. identified is the percentage of the hit that is identified, the accesion length is the length of the sequence corresponding to the accession number, and the accesion number is the number of the hit.

\subsubsection*{Based on the knowledge you have on 16S rRNA gene sequence similarities what could the candidate species and subspecies be (write them down below)}
The most likely hit would be the following strains. They are listed from most percent identified to least percent identified (ranging from 100-98.81\%).:
\begin{itemize}
    \item \textit{Bacillus subtilis} subsp. \textit{subtilis} BSP2\_43490
    \item \textit{Bacillus subtilis} CDA59\_06275
    \item \textit{Bacillus amyloliquefaciens} BN2127\_JRS5\_00115
    \item \textit{Bacillus subtilis} subsp. \textit{subtilis} BSP2\_43510
    \item \textit{Bacillus subtilis} subsp. \textit{subtilis} BSP2\_41070
\end{itemize}