\chapter{Litterature resumees}

\section{1. Lecture}

\subsection{Article 1 - Fermented Foods as Experimentally
Tractable Microbial Ecosystems}
\subsubsection*{Introduction}
The study of microbial communities faces significant challenges, largely due to their complexity and the difficulty in understanding how these multi-species communities are assembled, organized, and function. As a result, many aspects of microbial ecology remain unexplored.

The article highlights that the vast diversity and complexity of microbial communities are key factors contributing to our limited understanding. Additionally, the difficulty in isolating the majority of species from natural environments and cultivating them in vitro presents another notable challenge \cite*{L1-FermentedFoods}.

One approach to addressing these challenges is the development of simplified and controlled environments to study microbial communities. These controlled systems allow for reproducible results through measurable substrates, microbial growth, and community formation \cite*{L1-FermentedFoods}.

Several models with these properties have already been developed. These range from synthetic mixtures of microbial strains to more complex models composed of culture-able strains from free-living or host-associated communities, as well as naturally occurring communities that have been extensively sampled or perturbed in situ \cite*{L1-FermentedFoods}.

\subsubsection*{Fermented Foods as Experimentally Tractable
Ecosystems}
Fermented foods have been around for thousands of years and are an excellent example of how humans have optimized conditions to promote various properties of foods. Here are some of the main reasons why fermented foods have been produced for so long \cite*{L1-FermentedFoods}:
\begin{highlight}
    \begin{itemize}
    \item Good preserving effects
    \item Flavour can be improved
    \item Aroma can be enhanced
    \item Texture can be changed
    \end{itemize}
\end{highlight}

Some factors for manipulating microbial growth are listed below \cite*{L1-FermentedFoods}:
\begin{highlight}
    \begin{itemize}
    \item Temperature
    \item Salinity
    \item Moisture
    \end{itemize}
\end{highlight}

\vspace*{1em}
The article describes some micorbial communities of fermented foods (MCoFFs) and some examples of these, which will be listed bellow \cite*{L1-FermentedFoods}:
\begin{highlight}
    \begin{itemize}
    \item Multi-species biofilms associated with surfaces 
    \subitem e.g. cheese rinds
    \vspace*{0.5em}

    \item Suspended biofilms in liquid
    \subitem e.g. kombucha, kefir, and vinegar
    \vspace*{0.5em}

    \item Dispersed growth in liquid
    \subitem e.g. lambic beers, natural wines, and yogurt
    \vspace*{0.5em}

    \item Semi-solid substrates
    \subitem e.g. kimchi and miso
    \end{itemize}
\end{highlight}

Many MCoFFs have been studied in detail, and the article highlights that some of these communities have been shown to have bacterial- and fungi species that have co-evolved over time. The microbial diversity within and across fermented foods is vast, and the article emphasizes that these communities are excellent models for studying microbial ecology. In the article, cheese rind (the MCoFFs on the surface of the cheese) was used as an example of the characterization and development of MCoFFs, since on the surface of cheese the microbial communities form as it ages \cite*{L1-FermentedFoods}.

\subsubsection*{Using Fermented Foods to Link Patterns, Processes, and Mechanisms of Microbial Community Assembly}

In order to understand how species come together to form a community within a given environment microbiologists have developed several approaches for both plant- and animal communities. The ones named in the article is listed below \cite*{L1-FermentedFoods}: 

\begin{highlight}
    \begin{itemize}
        \item \textbf{Dispersal}
        \subitem Contributions from the amount and timing of microbial propagules colonizing a habitat
        \vspace*{0.3em}

        \item \textbf{Biotic selection}
        \subitem Interactions between species
        \vspace*{0.3em}

        \item \textbf{Abiotic selection}
        \subitem Interactions between a species and the environment
        \vspace*{0.3em}

        \item \textbf{Drift}
        \subitem Stochastic (random) changes in the relative abundances of species within communities
        \vspace*{0.3em}

        \item \textbf{Diversification}
        \subitem Evolution of new species within communities
    \end{itemize}
\end{highlight}

MCoFFs are ideal for linking microbial patterns with ecological processes. For instance, many fermented foods exhibit clear microbial successions where early colonizers are replaced by other species. This process can be studied to understand how environmental changes and species interactions drive these transitions. New techniques like RNA sequencing and transposon mutagenesis have proven valuable in uncovering the molecular mechanisms behind these interactions. Additionally, tools like imaging mass spectrometry are enhancing our understanding of how microbial species communicate chemically \cite*{L1-FermentedFoods}.

\subsubsection*{Taste of Place? Microbial Biogeography of Fermented Foods}

Though there are many different types of MCoFFs, many share some similarities. If a similar MCoFFs is used in two different cheese factories in two different locations, the microbial communities will be different, and with some distinct properties which can affect the texture, taste or aroma composition of the final product. This also enables the study of microbial biogeography, which is the study of how microbial communities are distributed across different locations. MCoFFs can therefore help linking patterns in microbial diversity from place to place. This is possible because each community has clear species abundance distributions \cite*{L1-FermentedFoods}.

\subsubsection*{Microbial Evolution in a Community Context}

MCoFFs offer valuable systems for studying both ecological processes and microbial evolution. Microorganisms used in beer, wine, dairy, and sake production have been useful in understanding how microbes adapt to fermentation environments. Research suggests that these organisms often lose genes not needed in nutrient-rich settings, gain new traits through gene transfer, and adjust their metabolism to fit the fermentation niche \cite*{L1-FermentedFoods}. 

Looking ahead, linking ecological and evolutionary processes could provide further insights. For example, studying long-term microbial communities could help explore how species interactions influence evolution. Experimentally testing coevolution within these communities could offer useful data on microbial adaptation over time \cite*{L1-FermentedFoods}.


\subsubsection*{Translation to Other Microbial Communities
and Potential Applications}

The study of microbial community assembly in traditional fermented foods (MCoFFs) can significantly enhance food quality and safety. Insights from these ecosystems can be applied to other microbial communities, such as the human microbiome. For example, the competitive mechanisms of \textit{Lactobacillus reuteri}, found in both sourdough and human gut microbiomes, illustrate how patterns observed in MCoFFs can be relevant to other systems \cite*{L1-FermentedFoods}.

Research on cheese rind microbial diversity has revealed similarities with the human skin microbiome, suggesting that moisture is a major driver of surface biofilms in both environments. This indicates that ecological selection processes may be conserved across different microbial communities \cite*{L1-FermentedFoods}.

MCoFFs can directly impact human health as they are consumed and can interact with the human gut microbiome. While the probiotic effects of simplified MCoFFs like yogurt have been extensively studied, the broader microbial diversity in MCoFFs could introduce additional beneficial microbes and metabolites to the human digestive system. Studies have shown that MCoFFs can remain viable during passage through the human digestive tract \cite*{L1-FermentedFoods}.

Furthermore, MCoFFs offer valuable opportunities for designing synthetic microbial communities for applications in medicine, industry, and agriculture. Understanding the interactions within these communities can inform the construction of synthetic microbial ecosystems. By studying pre-existing MCoFFs, researchers can learn about design principles and explore the potential for combining microbes from different MCoFFs into new, functional compositions \cite*{L1-FermentedFoods}. 

Ultimately, research on food microbial communities could result in safer, more delicious foods and contribute to the development of essential principles for microbial community design \cite*{L1-FermentedFoods}.

\vspace{1em}

\subsection{Article 2 - Review: Diversity of Microorganisms in global fermented foods and beverages}

\subsubsection*{Introduction}
The article describes how rice with fermented and non-fermented legumes is a staple diet in many counties in Asia. Wheat/barley-based breads/loaves followed bt milk and fermented milk products, meat and fermented meats are more common in the western world (West Asia, Europe and North America). The staple diet in Africa and South America comprise of sorghum and maize with wild legume seeds, meat and milk products \cite*{L1-DiversityMicro}. 
Consortia of microorganisms are found naturally in uncooked plant and animal materials, utensils, food containers, earthen pots and in general in the environment. When producing fermented foods, it is normal to introduce a starter culture, which contains functional microorganisms \cite*{L1-DiversityMicro}. The microorganisms in fermented food has converted the chemical composition of the raw food, which results in a change in the nutritional value of the food, making it more enriched \cite*{L1-DiversityMicro}.

\subsubsection*{Microorganisms in fermented foods}

\textbf{Lactic acid bacteria (LAB)} are one of the most important groups of microorganisms in fermented foods. They are used in the production of fermented foods and beverages. Some of the major genra of the LAB are listed below \cite*{L1-DiversityMicro}:
\begin{highlight}
    \begin{multicols}{3}
        \begin{itemize}
            \item \textit{Alkalibacterium}
            \item \textit{Carnobacterium}
            \item \textit{Enterococcus}
            \item \textit{Lactobacillus}
            \item \textit{Lactococcus}
            \item \textit{Leuconostoc}
            \item \textit{Oenococcus}
            \item \textit{Pediococcus}
            \item \textit{Streptococcus}
            \item \textit{Tetragenococcus}
            \item \textit{Vagococcus}
            \item \textit{Weissella}
        \end{itemize}
    \end{multicols}
\end{highlight}

\textbf{\textit{Bacillus}} has ben found in alkaline-fermented foods in Asia anf Africa. The species of \textit{Bacillus} that are found in abundance in legume-based fermented foods are listed bellow \cite*{L1-DiversityMicro}:

\begin{highlight}
    \begin{multicols}{3}
        \begin{itemize}
            \item \textit{Bacillus amyloliquefaciens}
            \item \textit{Bacillus circulans}
            \item \textit{Bacillus coagulans}
            \item \textit{Bacillus firmus}
            \item \textit{Bacillus licheniformis}
            \item \textit{Bacillus megaterium}
            \item \textit{Bacillus pumilus}
            \item \textit{Bacillus subtilis}
            \item \textit{Bacillus subtilis variety natto}
            \item \textit{Bacillus thuringiensis}            
        \end{itemize}
    \end{multicols}
\end{highlight}

There has been reported several species of \textit{Kocuria, Micrococcus} and \textit{Staphylococcus} in fermented foods. The species of \textit{Kocuria} are found in abundance in fermented milk products, fermented sausages, meat-, and fish products \cite*{L1-DiversityMicro}.

\vspace{1em}

\textbf{Yeasts} are also associated with fermentation of foods and alcoholic beverages. The yeasts named in the article are listed bellow \cite*{L1-DiversityMicro}:
\begin{highlight}
    \begin{multicols}{4}
        \begin{itemize}
            \item \textit{Brettanomyces}
            \item \textit{Candida}
            \item \textit{Cryptococcus}
            \item \textit{Debaryomyces}
            \item \textit{Dekkera}
            \item \textit{Galactomyces}
            \item \textit{Geotrichum}
            \item \textit{Hansenula}
            \item \textit{Hanseniaspora}
            \item \textit{Hyphopichia}
            \item \textit{Issatchenkia}
            \item \textit{Metschnikowia}
            \item \textit{Saccharomyces}
            \item \textit{Pichia}
            \item \textit{Kazachstania}
            \item \textit{Rhodotorula}
            \item \textit{Saccharomycodes}
            \item \textit{Kluyveromyces}
            \item \textit{Rhodosporidium}
            \item \textit{Saccharomycopsis}
            \item \textit{Schizosaccharomyces}
            \item \textit{Sporobolomyces}
            \item \textit{Torulaspora}
            \item \textit{Torulopsis}
            \item \textit{Trichosporon}
            \item \textit{Yarrowia}
            \item \textit{Zygosaccharomyces}                  
        \end{itemize}
    \end{multicols}
\end{highlight}

\textbf{Filamentous molds} are also found in fermented foods. They hold a major role in various fermented products in resprect of enzyme production and in the degradation of anti-nutritive factors. The listed filamentous molds are listed bellow \cite*{L1-DiversityMicro}. 

\begin{highlight}
    \begin{multicols}{4}
        \begin{itemize}
            \item \textit{Actinomucor}
            \item \textit{Amylomyces}
            \item \textit{Aspergillus}
            \item \textit{Monascus}
            \item \textit{Mucor}
            \item \textit{Neurospora}
            \item \textit{Paracilomyces}
            \item \textit{Penicillium}
            \item \textit{Rhizopus}
            \item \textit{Ustilago}
        \end{itemize}
    \end{multicols}
\end{highlight}

\subsubsection*{Taxonomic tools for identification of microorganisms from fermented foods}

A commonly used method for profiling both culturable and non-culturable microbial populations from fermented foods is DNA extraction. The technique is based on the separation of PCR-amplified DNA fragments by denaturing gradient gel electrophoresis (DGGE). This method is both used for profiling bacterial populations and yeast populations in fermented foods. Also, in fermented foods a combination of Propidium MonoAzide (PMA) treatment, which is done before DNA-extraction, and molecular quantifying methods can be used to enumerate the viable microorganisms accurately \cite*{L1-DiversityMicro}.
Random amplification of polymorphic DNA (RAPD) is a typing method based on the genomic DNA fragment profiles amplified by rep-PCR, and is commonly used for differentiation of LAB strains from fermented foods and other bacteria or yeast in the consortia. This technique is able to differentiate between strains of the same species \cite*{L1-DiversityMicro}.
Amplified fragment length polymorphism (AFLP) is another technique which is used to identify and discriminate LAB strains. The technique is based on the selective amplification and separation of genomic restriction fragments \cite*{L1-DiversityMicro}.

DGGE techniques and temperature gradient gel electrophoresis (TGGE) are used to study the microbial diversity in fermented foods. The techniques are based on the separation of PCR-amplified 16s rDNA and 26s rDNA gene fragments by denaturing gradient gel electrophoresis. The techniques are used to study the microbial diversity in fermented foods. DGGE is quite time consuming and is not able to determine relative abundance of the dominant species, nor is it able to distinguish between viable and non-viable cells \cite*{L1-DiversityMicro}.

Next generation sequencing (NGS) are effective tools for studying the microbial diversity in fermented foods. The techniques are able to sequence all the genetic material present in a sample, providing a comprehensive view of the microbial community's diversity and functional potential. These techniques require extensive training and a well-equipped molecular laboratory \cite*{L1-DiversityMicro}.

In table \ref{tab:NGS_techniques} some of the NGS which is listed in the text are described. These NGS are used to study the microbial diversity in fermented foods \cite*{L1-DiversityMicro}.


\begin{table}[H]
    \centering
    \caption{Some NGS techniques with their respective key-points}
    \label{tab:NGS_techniques}
    \begin{tabularx}{\textwidth}{p{0.33\textwidth}|p{0.33\textwidth}|p{0.33\textwidth}}
        \textbf{Metagenomics} & \textbf{Phylobiomics} & \textbf{Metatranscriptomics} \\
        \hline
        \begin{itemize}[left=0pt]
            \item Sequences all the genetic material present in a sample, providing a comprehensive view of the microbial community's diversity and functional potential.
            \item Allows the study of microbial communities without the need to culture individual species, overcoming limitations of traditional microbiology methods.
            \item By sequencing entire genomes, metagenomics can identify functional genes involved in processes like metabolism, antibiotic resistance, or nutrient cycling.
        \end{itemize} &
        \begin{itemize}[left=0pt]
            \item Is primarily concerned with analyzing the evolutionary relationships between microbial species, often using 16S rRNA or other conserved genes.
            \item It helps characterize the diversity and phylogenetic structure of microbial communities, determining how different species are related within an ecosystem.
            \item This technique is useful for identifying species and classifying them taxonomically based on their evolutionary lineage.
        \end{itemize} &
        \begin{itemize}[left=0pt]
            \item Focuses on sequencing the RNA transcripts present in a microbial community, providing a snapshot of active genes and gene expression levels.
            \item Reveals what genes are actively being expressed in a community at a given time.
            \item It helps assess how microbial communities respond to environmental changes, stress, or disease by analyzing shifts in gene expression.
        \end{itemize} \\
    \end{tabularx}
\end{table}

